package.json

```diff
@@ -113,9 +113,9 @@
     "@tiptap/extension-text": "2.6.6",
     "@tiptap/extension-text-align": "2.6.6",
     "@tiptap/extension-underline": "2.6.6",
-    "@tiptap/extension-text-style": "2.12.0",
-    "@tiptap/extension-color": "2.12.0",
-    "@tiptap/extension-font-family": "2.12.0",
+    "@tiptap/extension-text-style": "2.6.6",
+    "@tiptap/extension-color": "2.6.6",
+    "@tiptap/extension-font-family": "2.6.6",
```

Comment:
it looks like we may have to upgrade this due to a memory leak bug.. but we can do that separately from this PR

Reply:
I can update all of them to latest, I just wanted all of them to match

Reply:
Will do on a separate PR